lxqt-config-file-associations/lxqt-config-file-associations.desktop.in

```diff
@@ -9,3 +9,6 @@ Categories=Settings;DesktopSettings;Qt;LXQt;
 OnlyShowIn=LXDE;LXQt;
 
 #TRANSLATIONS_DIR=translations
+Name[pt]=Associação de ficheiros
```

Comment:
@smarquespt .desktop.in files are *not* to be edited.
To localize the .desktop file one should create and xxxxxxxxxx_pt.desktop in the translations dir.
Take https://github.com/lxde/lxqt-config/blob/master/lxqt-config-appearance/translations/lxqt-config-appearance_pt.desktop as an example.

I should write some documentation on this.